mage_ai/shared/utils.py

```diff
@@ -93,15 +96,17 @@ def convert_python_type_to_bigquery_type(python_type):
     return 'STRING'
 
 
-def convert_python_type_to_trino_type(python_type):
+def convert_python_type_to_trino_type(python_type, usr_data_types):
     if python_type is int:
         return 'BIGINT'
     elif python_type is float:
         return 'DOUBLE'
     elif python_type is bool:
         return 'BOOLEAN'
-    elif python_type is datetime:
+    elif python_type is datetime and usr_data_types is None:
         return 'TIMESTAMP'
+    elif python_type is datetime and usr_data_types is not None:
+        return f'TIMESTAMP({usr_data_types.get("timestamp_precision")})'
```

Comment:
if `usr_data_types` doesn't have `timestamp_precision` key, it'll throw error
also, we prefer using single quote in the code

Reply:
Modified in fd632852c12c167b2638456d98769f07b19c0d5a